test/generators/shop_model_generator_test.rb

```diff
@@ -29,6 +33,13 @@ class ShopModelGeneratorTest < Rails::Generators::TestCase
     end
   end
 
+  test "injects into application controller" do
+    run_generator
+    assert_file "app/controllers/application_controller.rb" do |controller|
+      assert_match "def shop", controller
+    end
```

Comment:
The Shop model I see, I assume is from the new App:

      class Shop < ActiveRecord::Base
        include ShopifyApp::Shop
        include ShopifyApp::SessionStorage
      end

Yet in my Home Controller, which inherits from AuthenticatedController, any call to shop is met with nil... 

     class HomeController < ShopifyApp::AuthenticatedController
         def index
             puts "Working on shop #{shop.shopify_domain}"
             @products = ShopifyAPI::Product.find(:all, params: { limit: 10 })
         end
     end

I guess that is why I am puzzled about where I went wrong.